conf/routes

```diff
@@ -9,3 +9,6 @@ GET     /hello                      controllers.HomeController.hello()
 
 # Map static resources from the /public folder to the /assets URL path
 GET     /assets/*file               controllers.Assets.versioned(path="/public", file: Asset)
+
+# API endpoints
+GET     /pass-bakery/status         controllers.APIController.getStatus()
```

Comment:
You're missing and end of line character in this file, please add it

Reply:
added a new line to routes as suggested